src/main/kotlin/com/sourcegraph/cody/chat/ChatSession.kt

```diff
@@ -179,6 +201,25 @@ private constructor(private val project: Project, newSessionId: CompletableFutur
     }
   }
 
+  @RequiresEdt
+  private fun addAssistantUsedContextToChat(message: ChatMessage) {
+    synchronized(this) {
+      val findResult = messages.indexOfLast { it.id == message.id }
```

Comment:
I think more idiomatic for kotlin would be using find and check if not null, but it's a nitpick.